server/routes/maintainingReports/coordinator.ts

```diff
@@ -167,30 +167,34 @@ export default class CoordinatorRoutes {
   confirmDeleteStatement: RequestHandler = async (req, res) => {
     const reportId = extractReportId(req)
     const { statementId } = req.params
+    const { removalRequest } = req.query
 
     const staffMember = await this.involvedStaffService.loadInvolvedStaff(reportId, parseInt(statementId, 10))
 
     const errors = req.flash('errors')
 
-    const data = { reportId, statementId, displayName: staffMember.name }
+    const data = { reportId, statementId, displayName: staffMember.name, removalRequest }
 
     res.render('pages/coordinator/confirm-statement-deletion.html', { errors, data })
   }
 
   deleteStatement: RequestHandler = async (req, res) => {
     const reportId = extractReportId(req)
     const { statementId } = req.params
-    const { confirm } = req.body
+    const { confirm, removalRequest } = req.body
 
     if (!confirm) {
       req.flash('errors', [{ href: '#confirm', text: 'Select yes if you want to delete this statement' }])
-      return res.redirect(`/coordinator/report/${reportId}/statement/${statementId}/confirm-delete`)
+      return removalRequest
+        ? res.redirect(paths.confirmStatementDelete(reportId, statementId, true))
+        : res.redirect(paths.confirmStatementDelete(reportId, statementId))
```

Comment:
we probably should have an extra test to cover this

Reply:
Done